jta-1_2-in-wildfly/src/test/java/org/jboss/narayana/quickstarts/jta/TestCase.java

```diff
@@ -22,8 +30,35 @@
  * @author <a href="mailto:gytis@redhat.com">Gytis Trikleris</a>
  */
 @RunWith(Arquillian.class)
+@ServerSetup(value = TestCase.ServerCmrSetup.class)
 public class TestCase {
 
+    public static class ServerCmrSetup implements ServerSetupTask {
```

Comment:
Could you rename this class with a name that clearly explains its purpose?

Reply:
Thanks @jmfinelli , my mistake. Fixing it.